shared/wallets/routes.js

```diff
@@ -7,49 +7,79 @@ import Container from './container'
 import ReceiveModal from './receive-modal/container'
 import ExportSecretKey from './export-secret-key/container'
 import TransactionDetails from './transaction-details/container'
+import WalletSettings from './wallet/settings/container'
+import {SetDefaultAccountPopup, RemoveAccountPopup, ReallyRemoveAccountPopup} from './wallet/settings/popups'
 import SendForm from './send-form/container'
 import ConfirmForm from './confirm-form/container'
+import Wallet from './wallet/container'
 
-const routeTree = makeRouteDefNode({
-  children: {
-    createNewAccount: {
-      children: {},
-      component: CreateNewAccount,
-      tags: makeLeafTags({layerOnTop: !isMobile}),
-    },
-    exportSecretKey: {
-      children: {},
-      component: ExportSecretKey,
-      tags: makeLeafTags({layerOnTop: !isMobile}),
-    },
-    linkExisting: {
-      children: {},
-      component: LinkExisting,
-      tags: makeLeafTags({layerOnTop: !isMobile}),
-    },
-    receive: {
-      children: {},
-      component: ReceiveModal,
-      tags: makeLeafTags({layerOnTop: !isMobile}),
-    },
-    sendReceiveForm: {
-      children: {
-        confirmForm: {
-          children: {},
-          component: ConfirmForm,
-          tags: makeLeafTags({layerOnTop: !isMobile}),
+const walletChildren = {
+  createNewAccount: {
+    children: {},
+    component: CreateNewAccount,
+    tags: makeLeafTags({layerOnTop: !isMobile}),
+  },
+  exportSecretKey: {
+    children: {},
+    component: ExportSecretKey,
+    tags: makeLeafTags({layerOnTop: !isMobile}),
+  },
+  linkExisting: {
+    children: {},
+    component: LinkExisting,
+    tags: makeLeafTags({layerOnTop: !isMobile}),
+  },
+  receive: {
+    children: {},
+    component: ReceiveModal,
+    tags: makeLeafTags({layerOnTop: !isMobile}),
+  },
+  settings: {
+    children: {
+      setDefaultAccount: {
+        children: {},
+        component: SetDefaultAccountPopup,
+        tags: makeLeafTags({layerOnTop: !isMobile}),
+      },
+      removeAccount: {
+        children: {
+          reallyRemoveAccount: {
```

Comment:
I'm not really happy with the route of `settings` > `removeAccount` > `reallyRemoveAccount`, and the popups stack, making a popup from a popup behave weirdly. Anyone have a solution to this? The ones I could think of are:
1. Don't have any popups be popups of other children, and navigate from a popup to another popup with `navigateTo`
2. Have 1 popup that manages what content of the popup to display, e.g., `LinkExisting`.

I'm leaning more towards 2, but want to know if there are other solutions.